internal/jetbrains/ideas_test.go

```diff
@@ -0,0 +1,104 @@
+//go:build testing
+
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+package jetbrains
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetIdeas(t *testing.T) {
```

Comment:
I suggest removing this test.

The `GetIdes` function simply returns a map of values. There's no need to test that. It just creates a burden for us to update when we bump IDE versions

Reply:
Thats true